SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the SunPy software library
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. github account, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombusperson
+
+The SunPy project is ultimately run by the whole community: users, developers, and members of the wider solar physics and scientific software communtiy, for the benifit of all.
+This document descibes how the project is run through delegation of this authority, primarily to the Steering Committee and the Project Members.
+Wherever possible all descisions taken should be done by consensus, considering input from both active contributors and the wider community.
+
+## Advisory Board
+The primary functions of the advisory board are: to advise the steering committee on management of the project, find funding for the project to ensure the core aims of the project can be met, nominate a Ombudsperson, and to ensure that the governance of the project is running as intended.
+
+* **Membership Limit**: The board shall consist of at least 5 members and no more than 10 members.
+Within these limits, the board may increase or decrease the number of people serving on the board, including for the purpose of staggering the terms of members.
+
+* **Term Length**: All members shall be elected to serve a maximum of a two-year term, however the term may be extended until a successor has been elected. 
+A member may be re-elected at the end of their term.
+
+* **Staggering**: Members terms shall be staggered so that approximately half the number of members will end their terms in any given year.
+The term of office shall be considered to begin January 1 and end December 31 of the second year in office, unless the term is extended until such time as a successor has been elected.
+
+* **Vacancies**: The board may fill vacancies or may appoint new members to fill a previously unfilled board position, subject to the maximum number of members.
```

Comment:
What's a situation the board would fill a vacancy instead of someone being elected?

Reply:
:thu

Reply:
mbs_up: